frontend/src/api/services/UserApi.ts

```diff
@@ -44,6 +45,19 @@ export class UsersApi {
     );
   }
 
+  //TODO: This method can potentially only be implemented on the frontend, we`re using 
+  //same obj, just passing an optional prop
+  public async createProspectUser(user: IUser): Promise<boolean>{
+    //return await this.apiClient.fetchWrapper("POST",`${BASE_ROUTE}/prospectUsers/create`,user)
+    user.is_prospect = true
+    return await this.apiClient.fetchWrapper("POST",`${BASE_ROUTE}/createProspect`,user);
+  }
+
+  public async getAllProspectUsers(): Promise<Array<IUser>>{
```

Comment:
Not a bad shortcut way to do this  but you must  have a dedicated call for this or modify getAllUsers with maybe some flags to indicate how to query db  ... take a look at fetchConditionalProjects api call or come with something dynamic because this won't work with pagination correctly once there is a large number of prospective user

Reply:
Modify getAllUsers like adding a boolean so we only fetch for prospect users? 
public async getAllUsers(getOnlyProspectUsers:boolean): Promise<Array<IUser>> {...}
We'll talk about it tomorrow

Reply:
Done